rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -85,6 +85,7 @@ def process(self, text, entities):
             for match in relevant_matches:
                 entity = {"start": match["start"],
                           "end": match["end"],
+                          "text": match["text"],
```

Comment:
yes 
{
extractor: "ner_duckling",
end: 33,
text: "tomorrow",
value: "2017-05-13T00:00:00.000+10:00",
entity: "time",
start: 25
}
value returns the duckling_dimensions object and text is what word(s) in the query duckling used

Ie "what is the weather like tomorrow"
duckling used 'tomorrow' (the text) to come up with the value 2017-05-13T00:00:00.000+10:00

for me, i added that line to see what Duckling was pulling out of the query to use, and also later in my application I can use the returned 'text' value to display